src/object_store/src/object/opendal_engine/fs.rs

```diff
@@ -20,11 +20,12 @@ use super::{EngineType, OpendalObjectStore};
 use crate::object::ObjectResult;
 impl OpendalObjectStore {
     /// create opendal fs engine.
-    pub fn new_fs_engine(root: String) -> ObjectResult<Self> {
+    pub fn new_fs_engine() -> ObjectResult<Self> {
         // Create fs backend builder.
         let mut builder = Fs::default();
 
-        builder.root(&root);
+        // Now fs engine is only used in CI, so we can hardcode root.
+        builder.root("/tmp/rw_ci");
```

Comment:
Currently fs engine is only used in out CI, and since this PR remove root in [OpenDALConfig](https://github.com/risingwavelabs/risingwave/blob/9acfac97f7e93b755c615f2550cdf87d809645b5/src/risedevtool/src/service_config.rs#L246), we can not set root in risedev.yml.